### PR TITLE
Deploy scripts work with minikube / minishift

### PR DESCRIPTION
**Pull request by jodyhuntatx**
updated README in openshift subdirectory documented minishift startup process
changed ADMIN_USERNAME environment variables to be more descriptive and updated scripts.

### CODE REVIEW

File: bootstrap.env [hunk 1]
```diff
@@ -1,18 +1,19 @@
 # defined in demo README.md
-export PLATFORM=kubernetes
-export MINIKUBE=true
-export CONJUR_VERSION=5
-export CONJUR_APPLIANCE_TARFILE=~/conjur-install-images/conjur-appliance-5.0.1.0.tar
-export CONJUR_APPLIANCE_IMAGE=conjur-appliance:5.0-stable
+export PLATFORM=openshift
+export MINIKUBE=false
```
Comment (jtuttle) [inline bootstrap.env]: Can we rename this to `MINIENV` or is there some reason to distinguish whether we're in minikube vs minishift? We can always check `PLATFORM && MINIENV`.

File: 8_print_config.sh [hunk 2]
```diff
@@ -6,17 +6,13 @@ set -euo pipefail
...
   conjur_master_route=$($cli get routes | grep conjur-master | awk '{ print $2 }')
   ui_url="https://$conjur_master_route"
 fi
 
-if [ $CONJUR_VERSION = '4' ]; then
-  password=$(rotate_api_key)
-elif [ $CONJUR_VERSION = '5' ]; then
-  password=$CONJUR_ADMIN_PASSWORD
-fi
+password=$CONJUR_ADMIN_PASSWORD
```
[the remark this answers is not in the document]
Comment (jodyhuntatx) [reply 8_print_config.sh]: Yes, users can have passwords AND API keys so both work. But the password is set in bootstrap.env and so easier to remember.

File: utils.sh [hunk 3]
```diff
@@ -1,10 +1,9 @@
 #!/bin/bash
 
 CONJUR_VERSION=${CONJUR_VERSION:-$CONJUR_MAJOR_VERSION} # default to CONJUR_MAJOR_VERSION if not set
-PLATFORM="${PLATFORM:-kubernetes}"  # default to kubernetes if not set
-MINIKUBE='0'
-MINISHIFT='0'
-
+if [[ "$PLATFORM" == "" ]]; then
```
[the remark this answers is not in the document]
Comment (jodyhuntatx) [reply utils.sh]: These were added by Dustin C. Not 100% sure why.

File: bootstrap.env (hunk 1 above)
Comment (jodyhuntatx) [reply bootstrap.env]: There's no need to distinguish and Minishift environment from an OpenShift environment. Minikube doesn't have an internal registry so is the only environment that needs such distinguishing.

**Pull request merged**